show/main.py

```diff
@@ -1057,5 +1057,19 @@ def ecn():
     proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True)
     click.echo(proc.stdout.read())
 
+#
+# 'reboot-cause' command ("show reboot-cause")
+#
+@cli.command('reboot-cause')
+def reboot_cause():
+    """Show cause of most recent reboot"""
+    PREVIOUS_REBOOT_CAUSE_FILE = "/var/cache/sonic/previous-reboot-cause.txt"
+    if not os.path.isfile(PREVIOUS_REBOOT_CAUSE_FILE):
+        click.echo("Unable to determine cause of previous reboot\n")
```

Comment:
>Unable to determine cause of previous reboot [](start = 20, length = 44)

You mentioned "Unexpected reboot" in the sample in PR comment.

Reply:
I have written another part of the mechanism in the base image (rc.local) which I intend to check into the buildimage repo along with this submodule update once this PR merges.